package.json

```diff
@@ -118,5 +131,8 @@
         "patchedDependencies": {
             "rrweb@2.0.0-alpha.13": "patches/rrweb@2.0.0-alpha.13.patch"
         }
+    },
+    "engines": {
+        "node": ">=20.1.0"
     }
```

Comment:
why do we need to bump the node version here?

we try to be pretty conservative with changes since it can be hard to predict consequence on other folks CI

is it this change that means we need the changes to eslint?

Reply:
(i'm pretty sure we can support at least >=18 since that's what i'm using locally, i haven't tested with 16)

Reply:
> The node version requirement is due to Cypress not running e2e tests with ESM unless it runs in node > 20.1.0. There may be another workaround, but in local, that was the only way I could get Cypress working.

ok, the version requirement bump is likely to be painful for an SDK. Node 18 isn't end of life until Apr 2025 

but equally don't want to leave folk with your problem hanging...

maybe we can do this another way - we have an examples folder - you can see for e.g. the nextjs folder at https://github.com/PostHog/posthog-js/tree/main/playground/nextjs

could you open another pr with a vite/remix example - (as small as possible 😊)

that should let us see the problem and help us keep it working in future

(i've never touched remix or i'd offer to make one - but i'm guessing you'll do a better job 💖 )